app/templates/buyers/award_details.html

```diff
@@ -65,7 +65,7 @@
       {%
           with
           url=url_for(".award_brief", framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id),
-          text="Back to previous page"
+          text="Previous page"
       %}
```

Comment:
https://award-prototype.cloudapps.digital/award/suppliers/award

Reply:
The prototype is inconsistent with what we are already doing on the marketplace which is using 'Back to previous page'.

It was previously agreed, probably on a different story, that we go with consistency for the moment.

Saying that, this is an app for buyers so they probably won't have seen the equivalent text in the brief responses app and so wouldn't actually see the inconsistencies.

Something to clarify with content just in case here.